zerver/openapi/zulip.yaml

```diff
@@ -9018,67 +9026,78 @@ paths:
                 last_update_id:
                   type: integer
                   description: |
-                    Specifies what presence data the client already has, allowing the
-                    server to only return presence data more recent than that.
+                    The identifier that specifies what presence data the client already
+                    has received, which allows the server to only return more recent
+                    user presence data.
 
-                    This parameter should be set to `-1` during initialization of the
-                    client in order to fetch all data, unless the client is obtaining
-                    initial presence metadata from the [`POST /register` endpoint](/api/register-queue).
+                    This should be set to `-1` during initialization of the client in
+                    order to fetch all user presence data, unless the client is obtaining
+                    initial user presence metadata from the
+                    [`POST /register`](/api/register-queue) endpoint.
 
-                    Afterwards, its value should be obtained by reading
-                    `presence_last_update_id` from the server's response to the previous
-                    request in order to implement incremental fetches of presence data.
+                    In subsequent queries to this endpoint, this value should be set to the
+                    most recent value of `presence_last_update_id` returned by the server
+                    in this endpoint's response, which implements incremental fetching
+                    of user presence data.
+
+                    When this parameter is passed, the user presence data in the response
+                    will always be in the modern format.
 
                     **Changes**: New in Zulip 9.0 (feature level 263). Previously, the
-                    server sent data for all users who had been active in the last two
-                    weeks unconditionally.
+                    server sent user presence data for all users who had been active in the
+                    last two weeks unconditionally.
                   example: 5
                 new_user_input:
+                  type: boolean
                   description: |
-                    Whether the user has interacted with the client since the previous
-                    presence request from this client, such as moving the mouse or using
-                    the keyboard.
+                    Whether the user has interacted with the client (e.g. moved the mouse,
+                    used the keyboard, etc.) since the previous presence request from this
+                    client.
 
-                    These data are used by the server to support certain [usage
+                    This data is used by the server to support certain [usage
```

Comment:
"This data is" reads awkwardly to me, but I think we should instead say "This value is used".